tutorials/global_illumination.md

```diff
@@ -0,0 +1,419 @@
+\page global_illumination Global Illumination
+
+This tutorial showcases how to enable real-time global illumination in Gazebo.
+
+<div style="text-align:center;">
+    \image html files/global_illumination/global_illumination.gif width=60%
+</div>
+
+## About global illumination
+
+Global illumination (GI) techniques account for illumination from indirect light reflections. When GI is enabled, objects not only receive direct light from light sources, but also from indirect light bouncing off of other surfaces, ensuring nice reflections and soft shadows.
```

Comment:
technically shadows are a different technique from GI so suggestion to rephrase the last bit:

```suggestion
Global illumination (GI) techniques account for illumination from indirect light reflections. When GI is enabled, objects not only receive direct light from light sources, but also from indirect light bouncing off of other surfaces, ensuring nice reflections and a more natural appearance.
```